engine-tests/src/tests/erc20_connector.rs

```diff
@@ -744,16 +895,27 @@ pub mod sim_tests {
             nep_141,
             erc20,
             aurora,
+            wnear,
+            wnear_erc20,
         }
     }
 
+    fn get_account_balance(account: &str, aurora: &AuroraAccount) -> u128 {
```

Comment:
This returns the amount of NEAR stored on the Aurora account on NEAR, correct?

Reply:
Yes, it is.